pkg/controllers/user/monitoring/projectHandler.go

```diff
@@ -168,6 +142,29 @@ func (ph *projectHandler) doSync(project *mgmtv3.Project, clusterName string) er
 	return nil
 }
 
+// doProjectGraphsSync creates the metric graphs per project for cluster monitoring
+func (ph *projectHandler) doProjectGraphsSync(project *mgmtv3.Project) error {
+	_, err := mgmtv3.ProjectConditionMetricExpressionDeployed.DoUntilTrue(project, func() (runtime.Object, error) {
```

Comment:
Please run this `doProjectGraphsSync` after the condition `NamespaceBackedResource` and `ProjectConditionInitialRolesPopulated` are set to true.